src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -409,9 +414,7 @@ public void simulationPeriodic() {
     turretSim.update(Robot.defaultPeriodSecs);
     turretAngleSim.setAngle(Math.toDegrees(turretSim.getAngleRads()));
     SmartDashboard.putNumber("Turret Angle", Math.toDegrees(turretSim.getAngleRads()));
-    simTurretPos.set(
-        Units.radiansToRotations(
-            turretSim.getAngleRads() / Constants.TurretConstants.TURRET_MOTOR_TURRET_RATIO));
+    simTurretPos.set(Units.radiansToRotations(turretSim.getAngleRads()));
```

Comment:
This needs to be converted from the turret gear to the low gear rotations (i.e. multiplied by LOW_GEAR_CAN_CODER_RATIO)

Reply:
Resolved in 6e163f1